management/errors_test.go

```diff
@@ -0,0 +1,42 @@
+package management_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/management"
+)
+
+// isResourceNotFoundTestCases is a set of structs comprising of the error
+// IsResourceNotFoundError should test and the expected result.
+var isResourceNotFoundTestCases = []struct {
+	err      error
+	expected bool
+}{
+	{
+		&management.AzureError{
```

Comment:
You might initialize these anonymous structs in a single line, would be easier to read. Not a problem though.